features/src/cuda/devcontainer-feature.json

```diff
@@ -15,11 +16,32 @@
         "11.5",
         "11.4",
         "11.3",
-        "11.2"
+        "11.2",
+        "11.1"
       ],
-      "default": "12.1",
+      "default": "12.2",
       "description": "Version of the CUDA Toolkit to install."
     },
+    "installcuDNN": {
```

Comment:
What about another option for `installMathLibs` for things like cublas, cufft, cusolver, cusparse? Or will those implicitly be excluded if we don't install cuDNN/cuTensor?